whois-api/src/main/java/net/ripe/db/whois/api/rdap/VCardBuilder.java

```diff
@@ -45,11 +45,6 @@ public VCardBuilder addAdr(final Set<CIString> addresses) {
         return this;
     }
 
-    public VCardBuilder addEmail(final Set<CIString> emails) {
```

Comment:
Maybe move the cleanup into a separate PR so it's better separated, it's confusing to do unrelated work in the same PR.

Reply:
I will

Reply:
https://github.com/RIPE-NCC/whois/pull/1350

Reply:
Thanks